ecosystem/python/sdk/CHANGELOG.md

```diff
@@ -4,12 +4,14 @@ All notable changes to the Aptos Python SDK will be captured in this file. This
 
 **Note:** Until we cut a 1.0.0 release, the Aptos Python SDK does not follow semantic versioning.
 
+## 0.6.2
+- Added custom header "x-aptos-client" to both sync/async RestClient
+
 ## 0.6.1
 - Updated package manifest.
 
 ## 0.6.0
 - Add token client.
 - Add support for generating account addresses.
-- Add support for http2
```

Comment:
?

Reply:
!! This was not the intention. I am not sure how that line got removed lol

Thank you for catching this, will add it back.